man/git-fork.md

```diff
@@ -21,6 +21,16 @@ git-fork(1) -- Fork a repo on github
 
   Remotes will use ssh if you have it configured with GitHub, if not, https will be used.
 
+  Create pull request for a project on GitHub via command line.
```

Comment:
Should update the "pull request" to "fork"

Reply:
Ack...good catch.  Fixed.